airbase-policy/src/main/resources/checkstyle/airbase-checks.xml

```diff
@@ -78,6 +78,14 @@
         <property name="format" value="^[ \t]*import com\.google\.common\.base\.MoreObjects;$" />
         <property name="message" value="com.google.common.base.MoreObjects should only be used with static imports" />
     </module>
+    <module name="RegexpMultiline">
+        <property name="format" value="^[ \t]*import com\.google\.common\.base\.Preconditions;$" />
+        <property name="message" value="com.google.common.base.Preconditions should only be used with static imports" />
```

Comment:
The caveat is that a caller class cannot define its own `checkArgument` method and then use Preconditions.
I think I am OK with this.

Reply:
Yes. I would find such custom method very misleading.

Reply:
This is actually a problem I've run into with String.format. I wanted to have a custom format() method that delegated to String.format, but that kind of usage is blocked. Very annoying.

Reply:
> I wanted to have a custom format() method that delegated to String.format, but that kind of usage is blocked. Very annoying.

I think one might say something opposite when noticed that the `format` got overriden. This surely depends on the case and person.

As with everything this is a tradeoff. https://github.com/prestosql/presto/pull/4733 showed that it was not a big deal in the code base. However, having this checkstyle rule makes a code review process a bit smoother which I find nice. Some people might find nit-picking comments a bit offensive. Thanks to this change we could avoiding discussions on obvious things.